vulture_os/toolkit/api_parser/crowdstrike/crowdstrike.py

```diff
@@ -214,7 +214,8 @@ def getAlerts(self, since, to):
             for alert in alerts:
                 finalRawAlerts += [alert]
 
-        if self.frontend.crowdstrike_request_incidents:
+        # self.frontend is None if test
+        if self.frontend is not None and self.frontend.crowdstrike_request_incidents:
```

Comment:
it's best not to use the frontend object directly, but to get the necessary info from it (when available) in the `data` passed in the `__init__()` function.
and then use these internal variables in the rest of the code